src/pages/calculator.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+// eslint-disable-next-line import/extensions
+import calculate from '../Logic/calculate';
```

Comment:
- [ ] As the previous reviewer said, there's an error while running your application due to the wrong file being imported.  Change `'../Logic/calculate'` with `'../logic/calculate'` means '../`L`ogic/calculate' with '../`l`ogic/calculate' => the capital letter `L` with small letter `l`. Kindly fix this :smiley: 
![Screenshot from 2022-12-08 21-27-50](https://user-images.githubusercontent.com/65068771/206551073-c17862b2-555a-41bc-8370-b0ca50efcb78.png)